en/docs/learn/configuring-totp.md

```diff
@@ -92,7 +92,7 @@ and security during the authentication for this application.
 ## Configuring the user profile
 
 1.  Login to the
-    [user portal](../../learn/user-portal) and go to **Personal info**.  
+    [my account](../learn/my-account.md) and go to **Personal info**.  
```

Comment:
```suggestion
    [my account](../learn/my-account) and go to **Personal info**.  
```

Reply:
../learn/my-account is not working i have checked but ../learn/my-account.md works